lib/indexer.rb

```diff
@@ -54,14 +54,17 @@ def merge_indexes
 
     # sort the tmp file into place
     #
-    # TMPDIR needs to be set to a partition with lots of space
+    # TMPDIR may need to be set to a partition with lots of space
     #
     # LC_ALL=C is needed to ensure that a byte sort is used
     # See https://specs.webrecorder.net/cdxj/0.1.0/#sorting
     puts('Sorting, this could take a while...')
     @log.info("sorting #{tmp_path} to #{index_path}")
-    cmd = "TMPDIR=/web-archiving-stacks/data/tmp/ LC_ALL=C sort #{tmp_path} --output \"#{index_path}\""
-    puts('Error while sorting') unless system(cmd)
+    cmd = "LC_ALL=C sort #{tmp_path} --output \"#{index_path}\""
+    unless system(cmd)
+      warn("Error while sorting #{tmp_path}")
+      exit
```

Comment:
Make sure to exit when there has been an error when sorting, or else we will remove the concatenation of all the CDXJ files.